docs/Documentation/tutorial.md

```diff
@@ -43,191 +45,6 @@ VIC model), such as:
      * Net upward sensible heat
      * ... ...
 *  ***Routed streamflow at specified stream locations***
-     * (can be any locations within the stream network, and the resolution is the same as grid cell, i.e., stream locations are specified by indicating the grid cell in which it falls) calculated by the routing model, daily or subdaily time step.
```

Comment:
I think what John wants to express here is the meteorological data generated by VIC disaggregator. Thus, I delete the air temperature, precipitation and wind speed.